posts/inside-rust/2022-05-04-CTCFT-may.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: CTCFT 2022-05-16 Agenda
```

Comment:
The date in the filename does not match the title.

Reply:
Thanks, I've fixed the filename to today's date, but that title represents the day of the event, so I think it should be fine like this :)